### ISSUE TITLE
How to avoid line-breaking or set preferred maxLength?

### ISSUE THREAD

**Issue by caub** [redhat-developer/vscode-yaml#88]
If I have a long line  (it's wrapped in vscode, so it's fine)

```yml
      - foobar "long string ....... but not so long and ok as one line"
```

vscode-yaml will reformat it in

```yml
      - >-
        foobar
        "long string ....... but not so long and ok as one line"
```

How to avoid this behavior?

Also this extension nukes comments, it's quite annoying

**Comment by JPinkney** [comment]
There currently are no settings for the formatter. Unfortunately there is not any good yaml formatters that I can find so I will need to re-write a better formatter in the future.

**Comment by JPinkney** [comment]
I've just found [this PR](https://github.com/prettier/prettier/pull/4563) on popular formatter. When it gets merged I will use that as the formatter instead.

**Comment by JPinkney** [comment]
This will be fixed on the 0.4.0 release happening in the next month or so.
